### DISCUSSION TITLE
Probably bricked my dongle

### DISCUSSION THREAD

**Discussion by Fiannax** [makerdiary/nrf52840-mdk-usb-dongle#102] [Q&A]
Hi, after an attemp to switch to UF2 Bootloader from Open Bootloader using this guide https://github.com/makerdiary/nrf52840-mdk-usb-dongle/tree/master/firmware/open_bootloader#:~:text=port%2Dname%3E-,Change%20to%20Open%20Bootloader%20from%20UF2%20Bootloader,-You%20can%20also i probably bricked my dongle because now my pc doesn't recognize it. It doesn't get in any mode if i double click the reset button nor if i connect it to USB while pushing the button. Is there anything I can do to unbrick it?

**Comment by fiberflow** [comment]
same here, how can I un-brick fix it? @caizelin

**Comment by fiberflow** [comment]
Flashing nrf52840_mdk_usb_dongle_open_bootloader_v1.2.0.hex with PyOCD works; however, flashing back to U2F with uf2_bootloader-nrf52840_mdk_usb_dongle-0.7.1-s140_6.1.1.hex does not works.

**Comment by Fiannax** [comment]
In the end I solved using a chep debug interface (https://it.aliexpress.com/item/1005003587741362.html?spm=a2g0o.order_list.order_list_main.72.41923696u5LcUo&gatewayAdapt=glo2ita) and following this guide (https://wiki.makerdiary.com/nrf52840-mdk-usb-dongle/programming/pitaya-link/)

**Reply by angelyordanov** [reply]
Did you managed to unbrick it with this debug interface or bought a new one?

**Reply by Fiannax** [reply]
I unbricked the dongle using the debug interface in the link

**Reply by angelyordanov** [reply]
10x. I've just ordered one as well. 🤞
